plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -73,12 +75,38 @@ export class EventPipelineRunner {
 
         // TODO: remove after successful rollout
         this.delayAcks = stringToBoolean(process.env.INGESTION_DELAY_WRITE_ACKS)
+
+        this.eventsToDropByToken = new Map()
+        process.env.DROP_EVENTS_BY_TOKEN_DISTINCT_ID?.split(',').forEach((pair) => {
```

Comment:
Have to be careful about distinct ids with `,` in?

Reply:
same as capture